core/src/main/scala/sttp/tapir/internal/package.scala

```diff
@@ -171,7 +171,7 @@ package object internal {
         case f: EndpointOutput.StatusCode[_] if f.documentedCodes.nonEmpty =>
           val entries = f.documentedCodes.keys.map(code => Some(code) -> Vector(f)).toList
           Right(entries)
-        case _: EndpointIO.Empty[_]     => Left(Vector.empty)
+        case e: EndpointIO.Empty[_]     => if (e.info.description.nonEmpty) Left(Vector(e)) else Left(Vector.empty)
```

Comment:
what happens if we remove this special-case entirely? Maybe that case here isn't needed at all, since we already handle this case in `EndpointToOperationResponse.scala`

Reply:
unfortunately when I remove this case entirely then 89 out of 183 tests in openapi-docs fail, would you like me to check why and try to fix them or just leave it as it is?

Reply:
Hmm that's weird, yes please check what's going on :)